crypto/cryptlib.c

```diff
@@ -355,4 +355,54 @@ int CRYPTO_memcmp(const void * in_a, const void * in_b, size_t len)
 
     return x;
 }
+
+/*
+ * For systems that don't provide an instruction counter register or equivalent
+ * we provide a best available clock.
+ * Return zero if no such timer is available.
+ */
+uint32_t OPENSSL_rdtsc(void)
+{
+# if defined(_AIX)
+    timebasestruct_t t;
+
+    read_wall_time(&t, TIMEBASE_SZ);
+    if (time_base_to_time(&t, TIMEBASE_SZ) == 0)
+        return (uint32_t)t.tb_low;
+    return 0;
+# elif defined(__sun) || defined(__hpux)
+    return (uint32_t)gethrtime();
+# elif defined(_WIN32)
+    LARGE_INTEGER t;
+
+    QueryPerformanceCounter(&t);
```

Comment:
Maybe the call should get remove in rand_drbg_get_additional_data() too?

Reply:
Matthias's patch does this.  I've added it.